docs/docs/docs/build-the-microsite.md

```diff
@@ -12,7 +12,28 @@ Once you have written your documents, you can build the microsite running this s
 sbt> makeMicrosite
 ```
 
-Internally, it'll sequentially run other tasks including its own, [`tut`](https://github.com/tpolecat/tut), and `makeSite` ([sbt-site](https://github.com/sbt/sbt-site)) tasks.
+Internally, it'll sequentially run other tasks including its own, [`tut`](https://github.com/tpolecat/tut) or [`mdoc`](https://scalameta.org/mdoc/), and `makeSite` ([sbt-site](https://github.com/sbt/sbt-site)) tasks.
+
+
+# Multiversion support
+
+Starting on version `1.0.0` you can build microsites including different versions of your library. This is available for the `light` theme, and you need to have [`git`](https://git-scm.com/) installed and available in your system. Most probably you already have it, but in other case please check your system docs, or go to the [git website](https://git-scm.com/download/) on how to proceed.
```

Comment:
Do you think it could be possible to use this feature in other themes down the road?

Reply:
Definitely!  It's only not being offered to the `pattern` theme at this time, along more features that are not ready yet, but I hope to bring them all over.

This PR was taking longer than I was expecting, so I reduce the scope a bit for this release.

Reply:
Makes sense